elide-core/src/main/java/com/yahoo/elide/core/Slf4jQueryLogger.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.core;
+
+import com.yahoo.elide.ElideResponse;
+import lombok.extern.slf4j.Slf4j;
+
+import java.security.Principal;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Slf4jQuery Logger Implementation for Elide. Overrides the default Noop QueryLogger
+ * Implementation of Elide
+ */
+@Slf4j
+public class Slf4jQueryLogger implements QueryLogger {
+    private volatile Map<UUID, Long> startTime;
+
+    public Slf4jQueryLogger() {
+        startTime = new ConcurrentHashMap<>();
+    }
+
+    @Override
+    public void acceptQuery(UUID queryId, Principal user, Map<String, String> headers, String apiVer, String apiQuery) {
+        synchronized (this) {
+            long start = System.currentTimeMillis();
```

Comment:
Rather than use a map, we might just log the start and end times.  The map could leak memory over time if we don't clean this up correctly.